api_docs/unmerged.d/ZF-392de9.md

```diff
@@ -0,0 +1,8 @@
+* [`POST /channel_folders/create`](/api/create-channel-folder): Added
+  a new endpoint for creating a new channel folder.
+
+* [`GET /events`](/api/get-events): An event with `type: "channel_folder"` is
+  sent to the current user when a channel folder is created.
```

Comment:
This framing makes it sound like only the current use gets notified?

Reply:
Oops! Fixed.